src/lib/forward.ts

```diff
@@ -110,6 +111,40 @@ export class EventForwarder {
         ...extra,
       });
 
+    // Value or Metadata updates require potential schema transforms
+    const notifyNodeOnValueOrMetadataUpdates = (
+      node: ZWaveNode,
+      event: string,
+      args: any
+    ) => {
+      let newArgs: any;
+      this.clients.clients.forEach((client) => {
+        // Copy arguments for each client so transforms don't impact all clients
+        Object.assign(newArgs, args);
+        if ("prevValue" in newArgs) {
+          schemaTransformValueMetadata(
+            newArgs.prevValue.metadata,
+            client.schemaVersion
+          );
+        }
+        if ("newValue" in newArgs) {
+          schemaTransformValueMetadata(
+            newArgs.newValue.metadata,
+            client.schemaVersion
+          );
+        }
+        if ("metadata" in newArgs) {
+          schemaTransformValueMetadata(newArgs.metadata, client.schemaVersion);
+        }
+        this.sendEvent(client, {
+          source: "node",
+          event,
+          nodeId: node.nodeId,
+          ...{ newArgs },
```

Comment:
I don't think this is the same, right? When I look at the old code, it had `{ args }`, which meant that there was a single `args` key that contains the value. That would be equivalent here to:
```suggestion
          { args: newArgs },
```

Reply:
This is causing CI to fail. Does `args` need to be in quotes?

Reply:
That didn't work either 🤔

Reply:
Oh I think it's because we lost the spread operator

Reply:
OK I see, we can avoid creating an object and just pass the param directly